src/features/projects/screens/Projects.tsx

```diff
@@ -27,7 +27,7 @@ interface Props {
 }
 
 const ProjectSnippet = dynamic(
-  () => import('../../projectsV2/components/ProjectSnippet'),
+  () => import('../../projectsV2/ProjectSnippet'),
```

Comment:
Why are we using the ProjectSnippet from v2 here, instead of the original project snippet?